src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        String randomColor = ColorSupplier.getRandomColor();

```

Comment:
Avoid using static methods from `ColorSupplier` as per the checklist. Instead, create an instance of `ColorSupplier` and call `getRandomColor()` on that instance.